cmd/noisd/init.go

```diff
@@ -111,6 +111,10 @@ func InitCmd(mbm module.BasicManager, defaultNodeHome string) *cobra.Command {
 			// as non-browser clients
 			config.RPC.CORSAllowedOrigins = []string{"*"}
 
+			// Peer settings
+			config.P2P.MaxNumInboundPeers = 80
+			config.P2P.MaxNumOutboundPeers = 40
```

Comment:
Is this a good value? I put it a bit randomly from validator's recommendation into the docs. The Tendermint defaults are super low.

Reply:
I'd go with a balanced value perhaps 70/70